.build-tools/go.mod

```diff
@@ -17,3 +17,5 @@ require (
 	github.com/xeipuuv/gojsonreference v0.0.0-20180127040603-bd5ef7bd5415 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 )
+
+replace github.com/dapr/dapr => github.com/1046102779/dapr v0.0.0-20221021130037-635b70c24259
```

Comment:
Nit: can you add a comment saying `// TODO: TEMPORARY ONLY`?

Reply:
Considering that using `replaceruntime-all` and `dropreplaceruntime-all` command to change go.mod. If add a comment:  `// TODO: TEMPORARY ONLY`, I'm worried about forgetting to remove it. 


See if you still need to do this?